infra-as-code/bicep/orchestration/hubPeeredSpoke/hubPeeredSpoke.bicep

```diff
@@ -0,0 +1,206 @@
+/*
+SUMMARY: This module acts as an orchestration module that create and configures a spoke network to deliver the Azure Landing Zone Hub & Spoke architecture
+DESCRIPTION:
+
+AUTHOR/S: KiZach
+VERSION: 1.0.0
+*/
+
+// **Parameters**
+// Generic Parameters - Used in multiple modules
+@description('The region to deploy all resoruces into. DEFAULTS TO = northeurope')
+param parLocation string = 'westeurope'
```

Comment:
Can we default this to the deployment location or the resource group location instead of a fixed location, like we do in other modules?

Please update the `README.md` also 👍

Reply:
Resource group creation is part of the module so I think we need this parameter right?

Reply:
Correct we need the parameter but it's default value should be changed as per previous comment. 👍

Reply:
Updated in newest commit